pkgs/top-level/all-packages.nix

```diff
@@ -585,6 +585,12 @@ in
     stdenv = clangStdenv;
   };
 
+  aflplusplus = callPackage ../tools/security/aflplusplus {
+    stdenv = clangStdenv;
+    python = python37;
```

Comment:
Why pin to 3.7? Is there a reason for that?

Reply:
I'm having to do a bit of archaeology to figure this out, but the readme for the python mutator at the time made specific mention of python 3.7: https://github.com/AFLplusplus/AFLplusplus/blob/939721e2cbcf14f0f9577575c09e16be2ced25fe/python_mutators/README#L4 so I must have assumed some version sensitivity.